src/useForm.tsx

```diff
@@ -69,7 +69,10 @@ function getInitialState<T>(
     forEach<FieldDefinition<any>>(
       fieldDefs,
       (path, { rules, default: defaultFieldValue, __type }) => {
-        const value = defaultFieldValue || get(defaultValue, path)
+        const value =
+          defaultFieldValue === ''
```

Comment:
Oh...we should actually explicitly check for `!== undefined` here as we could potentially have other falsey values like `false` :). Will change.